rs/registry/admin/src/main.rs

```diff
@@ -3938,21 +3956,19 @@ async fn main() {
         }
         SubCommand::ProposeToChangeNnsCanister(cmd) => {
```

Comment:
Agree. It's very hard to change though, since we would have to either create inconsistency between the command name and the struct name, or we have to track down people who uses the command in the script and change there.